### DISCUSSION TITLE
标题中的数字貌似不是黑体

### DISCUSSION THREAD

**Discussion by LiuChang0717** [nuaatug/nuaathesis#109] [Q&A]
作者们好，我在使用硕士模板的时候发现，编译出来的PDF文档中标题里的数字似乎并不是黑体，或者说是和Word文档中不完全一样的黑体，请问这样的问题如何解决？

这是未修改的模板中的字体（字体似乎是“方正黑体简体”？）：
![image](https://github.com/nuaatug/nuaathesis/assets/63938570/490c0a9a-7710-49b6-9125-fa6f29153b03)
这是我添加了自己论文内容后编译了的字体：
![image](https://github.com/nuaatug/nuaathesis/assets/63938570/c5c21298-ea1f-4a5f-83c1-be492f7143e3)
这是学校的模板用Word打开时的字体：
![image](https://github.com/nuaatug/nuaathesis/assets/63938570/b22f38f7-13b9-485a-940f-c1049d3d92d2)

可以明显看出数字的字体是不完全一样的，请问是否有比较简单的解决方法？

期待您们的回复。

**Comment by LiuChang0717** [comment]
另外补充一下，我使用的是overleaf在线编辑器

**Reply by yzwduck** [reply]
可能这就是问题所在. Release 包里的 PDF 文件是在 Windows 上编译的, ctex 会使用 Windows 的中文字体. 而 overleaf 应该是在 Linux 系统上编译的, 我猜 ctex 用了 fandol 字体.
如果在意字体差异的话, 可以考虑在 Windows 上编译最后用于提交的版本.

**Reply by LiuChang0717** [reply]
谢谢您的回答，我发现nuaathesis.cls文件中的\newcommand\nuaa@font@title{\sffamily\heiti}似乎控制了标题中的字体，将英文字体设定为一种无衬线字体，中文字体设置为黑体。所以是否可以通过修改这一行来实现英文字体也为黑体？不过我目前还么找到解决方法。

**Reply by yzwduck** [reply]
好像有点困难, ctex/xecjk 应该是根据字符集来决定使用中文字体还是西文字体. 如果要对英文使用黑体的话, 要翻阅 ctex/xecjk 文档, 看它有没有提供相关的设置.
我稍微看了一下这两文档, 没能一下子找到相关设置 (估计即使能找到设置调整英文字体, 效果是全局生效的, 会影响正文部分).
